contrib/mac/app/Entitlements.plist

```diff
@@ -4,6 +4,8 @@
 <dict>
 	<key>com.apple.security.automation.apple-events</key>
 	<true/>
+	<key>com.apple.security.cs.get-task-allow</key>
```

Comment:
It should be `com.apple.security.get-task-allow` (delete the `.cs`).

Reply:
Perhaps you can open a PR with the fix?